test/unit/uk/gov/hmrc/apisubscriptionfields/service/SubscriptionFieldsServiceSpec.scala

```diff
@@ -29,71 +29,69 @@ class SubscriptionFieldsServiceSpec extends UnitSpec with SubscriptionFieldsTest
   private val mockUuidCreator = mock[UUIDCreator]
   private val service = new SubscriptionFieldsService(mockSubscriptionFieldsIdRepository, mockUuidCreator)
 
-  private val SomeOtherFields = Map("f3" -> "v3", "f2" -> "v2b")
-
   "A RepositoryFedSubscriptionFieldsService" should {
-    "return an None when no entry exist in the repo when get by application client id is called" in {
+    "return None when no entry exist in the repo when get by application client id is called" in {
       (mockSubscriptionFieldsIdRepository fetchByClientId  _) expects fakeRawClientId returns List()
 
       val result = await(service.get(FakeClientId))
 
       result shouldBe None
     }
 
-    "return an Some response when entry exists in the repo when get by application client id is called" in {
+    "return Some response when entry exists in the repo when get by application client id is called" in {
       val subscriptionFields1 = createSubscriptionFieldsWithApiContext()
       val subscriptionFields2 = createSubscriptionFieldsWithApiContext(rawContext = fakeRawContext2)
       (mockSubscriptionFieldsIdRepository fetchByClientId  _) expects fakeRawClientId returns List(subscriptionFields1, subscriptionFields2)
 
       val result = await(service.get(FakeClientId))
 
       result shouldBe Some(BulkSubscriptionFieldsResponse(fields = Seq(
-        SubscriptionFieldsResponse(id = "TODO: remove this field", fieldsId = SubscriptionFieldsId(subscriptionFields1.fieldsId), fields = subscriptionFields1.fields),
-        SubscriptionFieldsResponse(id = "TODO: remove this field", fieldsId = SubscriptionFieldsId(subscriptionFields2.fieldsId), fields = subscriptionFields2.fields)
+        SubscriptionFieldsResponse(clientId = subscriptionFields1.clientId, apiVersion = subscriptionFields1.apiVersion, apiContext = subscriptionFields1.apiContext, fieldsId = SubscriptionFieldsId(subscriptionFields1.fieldsId), fields = subscriptionFields1.fields),
+        SubscriptionFieldsResponse(clientId = subscriptionFields2.clientId, apiVersion = subscriptionFields2.apiVersion, apiContext = subscriptionFields2.apiContext, fieldsId = SubscriptionFieldsId(subscriptionFields2.fieldsId), fields = subscriptionFields2.fields)
       )))
     }
 
-    "return None when no entry exist in the repo when get by composite id is called" in {
-      (mockSubscriptionFieldsIdRepository fetch _) expects FakeSubscriptionIdentifier returns None
+    "return None when no entry exists in the repo when get by subscription field is called" in {
```

Comment:
`return None when no entry exists in the repo`

Reply:
Done